gui/wxpython/projpicker_gui/gui.py

```diff
@@ -0,0 +1,752 @@
+"""
+This module implements the ProjPicker GUI using wxPython.
+"""
+
+import wx.lib.statbmp
+import io
+import threading
+import queue
+import textwrap
+import webbrowser
+
+import grass.projpicker as ppik
+if __package__:
```

Comment:
That `if __package__:` code is for running `projpicker.py` directly from the source directory without installation. I tried the PEP 366 code, but I was not able to run the script from where it is. For example, I installed projpicker `pip install projpicker` and made the above change in `site-packages/projpicker/projpicker.py` like this:
```python
if __name__ == "__main__" and __package__ is None:
    __package__ = "projpicker"

# https://stackoverflow.com/a/49480246/16079666
from .common import (BBox, coor_sep, pos_float_pat, bbox_schema,
                     bbox_columns, is_verbose, get_float)
from . import coor_latlon
from . import coor_xy
```
```bash
projpicker -h    # works

python site-packages/projpicker/projpicker.py -h      # doesn't work
Traceback (most recent call last):
  File "site-packages/projpicker/projpicker.py", line 48, in <module>
    from .common import (BBox, coor_sep, pos_float_pat, bbox_schema,
  File "/home/user/usr/local/lib/python3.7/site-packages/projpicker/projpicker.py", line 48, in <module>
    from .common import (BBox, coor_sep, pos_float_pat, bbox_schema,
ImportError: attempted relative import with no known parent package
```
If we never need to run `projpicker.py` from its installation directory, I think we just need relative imports as long as they are in the same directory or we only need absolute imports because these submodules are always installed together.